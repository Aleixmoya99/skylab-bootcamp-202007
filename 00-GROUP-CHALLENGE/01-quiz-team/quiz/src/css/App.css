.app {
  width: 100%;
}

body{
  background: black;
}