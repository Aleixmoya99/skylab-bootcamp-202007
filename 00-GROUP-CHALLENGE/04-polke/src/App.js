import React from 'react';
import HeaderNavbar from './components/Navbar/HeaderNabvar';
import Footer from './components/Footer/Footer';
import Landing from './components/Landing/Landing';
import RepoDetail from './components/RepoDetail/RepoDetai';
import UserDetail from './components/UserDetail/UserDetail';
import { Route, Switch } from 'react-router-dom';

function App() {
	return (
		<>
			<HeaderNavbar />
			<Switch>
				<Route path="/" exact component={Landing} />
				<Route path="/userDetail" component={UserDetail} />
				<Route path="/repoDetail" component={RepoDetail} />
			</Switch>
			<Footer />
		</>
	);
}

export default App;
