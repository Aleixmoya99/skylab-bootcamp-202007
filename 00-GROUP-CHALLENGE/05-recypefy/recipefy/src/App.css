* {
	margin: 0;
	padding: 0;
	font-family: Arial, Helvetica, sans-serif !important;
}

body {
	background-color: #d1eaa3;
	width: 100vw;
}

a {
	text-decoration: none;
}

h2 {
	font-family: Georgia, 'Times New Roman', Times, serif;
	color: rgb(75, 75, 165);
	font-size: 18px;
	text-transform: uppercase;
}

h3 {
	font-family: Georgia, 'Times New Roman', Times, serif;
	color: rgb(75, 75, 165);
}

h4 {
	font-family: Georgia, 'Times New Roman', Times, serif;
	color: rgb(75, 75, 165);
}

p {
	font-family: Georgia, 'Times New Roman', Times, serif;
	color: rgb(75, 75, 165);

}

.body__box {
	height: 35vh;
	width: 70%;
	background-color: #efee9d;
}