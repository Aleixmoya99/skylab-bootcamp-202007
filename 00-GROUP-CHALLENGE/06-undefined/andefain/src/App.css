body {
	font-family: 'Roboto', sans-serif;
	background-color: #101010;
}

.margin-bottom {
	margin-bottom: 150px;
}