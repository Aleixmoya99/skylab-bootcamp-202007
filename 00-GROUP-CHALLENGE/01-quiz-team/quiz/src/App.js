import React from 'react';
import './css/App.css';
import { Route, Switch } from 'react-router-dom';
import PageNotFound from './components/PageNotFound';
import Header from './components/Header';
import Footer from './components/Footer';
import LeaderBoard from './components/LeaderBoard';
import LoginScreen from './components/LoginScreen';
import Dashboard from './components/Dashboard';
import Question from './components/Question';
import Game from './components/Game';

function App() {
	return (
		<div className="app">
			<Header />
			<Switch>
				<Route path="/" exact component={Dashboard} />
				<Route path="/leaderboard" component={LeaderBoard} />
				<Route path="/login" component={LoginScreen} />
				<Route path="/question" component={Question} />
				<Route path="/themes/:themeSlug" component={Game} />
				<Route component={PageNotFound} />
			</Switch>
			<Footer />
		</div>
	);
}

export default App;
