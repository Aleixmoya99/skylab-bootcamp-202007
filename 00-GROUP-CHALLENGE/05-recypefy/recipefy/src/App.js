import React from 'react';
import { Route, Switch } from 'react-router-dom';
import Footer from './components/Footer';
import Header from './components/Header';
import Main from './components/Main';
import Profile from './components/Profile';
import RecipePage from './components/RecipePage';
import SearchResult from './components/SearchResult';
import UserFavourite from './components/UserFavourite';
import './App.css';
import Login from './components/Login';

function App() {
	return (
		<>
			<Header />

			<Switch>
				<Route path="/" exact component={Main} />
				<Route path="/profile" component={Profile} />
				<Route path="/recipe-page/:recipeId" component={RecipePage} />
				<Route path="/search-result" component={SearchResult} />
				<Route path="/userfavourite" component={UserFavourite} />
				<Route path="/login" component={Login}/>
			</Switch>

			<Footer />
		</>
	);
}

export default App;
