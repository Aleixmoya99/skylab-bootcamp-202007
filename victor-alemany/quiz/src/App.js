import React from 'react';
import './css/App.css';
import { Route, Switch } from 'react-router-dom';
import PageNotFound from './components/PageNotFound'
import Home from './components/Home'
import Header from './components/Header'
import Footer from './components/Footer'
import LeaderBoard from './components/LeaderBoard'
import LoginScreen from './components/LoginScreen'
import QuizComponent from './components/QuizComponent'


function App() {
	return (
		<div className="app">
			<Header />
			<Switch>
			<Route path="/" exact component={Home}/>
      <Route path="/leaderboard" exact component={LeaderBoard}/>
      <Route path="/login" exact component={LoginScreen}/>
			<Route component={PageNotFound} />
			</Switch>
      <QuizComponent />
      <Footer />
		</div>
	);
}

export default App;
