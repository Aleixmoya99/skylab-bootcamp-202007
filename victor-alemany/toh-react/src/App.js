import React from 'react';
import './App.css';
import HeroDetail from './components/HeroDetail';
import Header from './components/Header';
import HeroDashboard from './components/HeroDashboard';
import HeroList from './components/HeroList';
import { Route, Switch, Redirect } from 'react-router-dom';
import PageNotFound from './components/PageNotFound'

function App(props) {
	return (
		<main className = "main-container">
			<Header />
			<Switch>
			<Route path="/" exact component={HeroDashboard}/>
			<Route path="/hero" exact component={HeroList}/>
			<Route path="/hero/:heroId" component={HeroDetail} />
			<Redirect from="/heroes" to="/hero" />
			<Route component={PageNotFound} />
			</Switch>
		</main>
	);
}

export default App;