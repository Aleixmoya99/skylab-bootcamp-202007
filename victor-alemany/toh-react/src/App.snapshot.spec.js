import renderer from 'react-test-renderer'
import App from './App.js'
import React from 'react'


describe('App snapshot', ()=>{
    const tree = renderer.create(<App />);

    it('should match', ()=>{
        expect(tree.toJSON()).toMatchSnapshot();
    })
})