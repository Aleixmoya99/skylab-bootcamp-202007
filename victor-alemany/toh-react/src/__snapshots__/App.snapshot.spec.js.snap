// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot should match 1`] = `
<main
  className="main-container"
>
  <div>
    <h1>
      Tour of Heroes
    </h1>
  </div>
  <nav>
    <a
      aria-current="page"
      className="my-active-class"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      Dashboard
    </a>
     
     | 
    <a
      aria-current={null}
      href="/hero"
      onClick={[Function]}
    >
      Heroes
    </a>
     
     | 
    <a
      aria-current={null}
      href="/login"
      onClick={[Function]}
    >
      Login
    </a>
     
     | 
    <a
      aria-current={null}
      href="/heroes"
      onClick={[Function]}
    >
      Heroes
    </a>
  </nav>
  <div>
    <h1>
      Top Heroes
    </h1>
  </div>
  <div
    className="dashboard-item-container"
  />
</main>
`;
